src/store/actions/mainForm/index.ts

```diff
@@ -0,0 +1,6 @@
+import { ADD_DATA } from "store/types";
+
+export const mainFormData = (data: any) => ({
```

Comment:
This action suggests me that it should add data to store
so I think its name should be something like this
```suggestion
export const addUserInfoToReduxAction = (userInfo: any) => ({
```

Reply:
also @ZainShahbaz if you know the parameters of 'data' then please create interface and include that here